### PR TITLE
Load missing county names in ATP inbound payloads based on zip code

### PR DESCRIPTION
**Pull request by mdkaraman**
ME-180574281

### CODE REVIEW

File: components/financial_assistance/app/domain/financial_assistance/operations/transfers/medicaid_gateway/account_transfer_in.rb [hunk 1]
```diff
@@ -28,14 +28,34 @@ def call(params)
...
           end
 
+          private
+
           def county_finder(zip)
             ::BenefitMarkets::Locations::CountyZip.where(zip: zip)
           end
 
-          private
+          def load_missing_county_names(payload)
```
Comment (kristinmerbach) [inline components/financial_assistance/app/domain/financial_assistance/operations/transfers/medicaid_gateway/account_transfer_in.rb:37]: add a rescue

File: components/financial_assistance/app/domain/financial_assistance/operations/transfers/medicaid_gateway/account_transfer_in.rb [hunk 2]
```diff
@@ -397,7 +417,7 @@ def find_application(id)
 
           def fill_applicants_form(payload, application) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
             applications = payload["family"]['magi_medicaid_applications'].first
-            zips_with_missing_counties = []
+            # zips_with_missing_counties = []
```
Comment (kristinmerbach) [inline components/financial_assistance/app/domain/financial_assistance/operations/transfers/medicaid_gateway/account_transfer_in.rb:420]: remove commented line

**Review by kristinmerbach — CHANGES_REQUESTED** [review]
Looks good in general, just a couple v minor changes needed

File: components/financial_assistance/app/domain/financial_assistance/operations/transfers/medicaid_gateway/account_transfer_in.rb (hunk 2 above)
Comment (mdkaraman) [reply components/financial_assistance/app/domain/financial_assistance/operations/transfers/medicaid_gateway/account_transfer_in.rb]: Good catch!  Just removed this.

File: components/financial_assistance/app/domain/financial_assistance/operations/transfers/medicaid_gateway/account_transfer_in.rb (hunk 1 above)
Comment (mdkaraman) [reply components/financial_assistance/app/domain/financial_assistance/operations/transfers/medicaid_gateway/account_transfer_in.rb:37]: Added a rescue block to this method in the latest commit.

**Review by kristinmerbach — APPROVED** [review]
Looks good, thanks for getting this done!

**Pull request merged**